entries/jQuery.ajax.xml

```diff
@@ -281,24 +281,31 @@ jqxhr.always(function() {
         <code>readyState</code>
       </li>
       <li>
+        <code>responseXML</code> and/or <code>responseText</code> when the underlying request responded with xml and/or text, respectively
+      </li>
+      <li>
         <code>status</code>
       </li>
       <li>
         <code>statusText</code>
       </li>
-      <li><code>responseXML</code> and/or <code>responseText</code> when the underlying request responded with xml and/or text, respectively</li>
-      <li><code>setRequestHeader(name, value)</code> which departs from the standard by replacing the old value with the new one rather than concatenating the new value to the old one</li>
       <li>
-        <code>getAllResponseHeaders()</code>
+        <code>abort( [ statusText ] )</code>
+      </li>
+      <li>
+        <code>getAllResponseHeaders()</code> as a string
```

Comment:
Is it worth adding "as a string" here? The others don't talk about return types in general.